src/silx/io/dictdump.py

```diff
@@ -32,9 +32,9 @@
 import os.path
 import h5py
 try:
-    import pint
+    from pint import Quantity as PintQuantity
 except ImportError:
-    pint = None
+    PintQuantity = None
```

Comment:
I propose

```python
try:
    from pint import Quantity as PintQuantity
except ImportError:
    try:
        from pint.quantity import Quantity as PintQuantity
    except ImportError:
        PintQuantity = None
```

Also it is no longer needed to `import pint`.

Reply:
Done